Sources/Parley/Views/ParleyMessageView/ParleyMessageView.swift

```diff
@@ -195,6 +198,7 @@ final class ParleyMessageView: UIView {
         
         imageNameLabel.text = message.agent?.name
         nameLabel.text = message.agent?.name
+        nameLabel.adjustsFontForContentSizeCategory = true
```

Comment:
The name label was not changing fontSize when Dynamic Font Category was altered.